specification/cpim/resource-manager/readme.md

```diff
@@ -28,7 +28,7 @@ These are the global settings for the azureadb2c APIs.
 title: CPIMConfigurationClient
 description: CPIM Configuration Client 
 openapi-type: arm
-tag: package-2020-05-01-preview
+tag: package-2019-01-01-preview
```

Comment:
why to decrease the default version?

Reply:
The cli team is working on sdk generation for the 2019-01-01-preview version, not the 2020-05-01-preview. We're prioritization 2019-01-01-preview at this time.